torch/_utils.py

```diff
@@ -592,6 +592,25 @@ def dec(fun):
     return dec
 
 
+def render_call(fn, args, kwargs):
+    import torch._tensor_str
```

Comment:
I'm sure this file is already imported by default so that shouldn't be needed. And I don't expect this function to be called during import right?